builder/src/scss/package-list.scss

```diff
@@ -1,5 +1,121 @@
 @import "./theme/_variables";
 
+.lander-column-margins {
+    margin-top: 8%;
+    margin-bottom: 8%;
+}
+
+.mod-count-box > img {
+    max-width: 96px;
+    max-height: 96px;
+}
+
+.modspic {
+    max-width: 500px;
+    max-height: 500px;
+}
+
+.mod-manager-pic {
+    width: 12rem;
+    height: 12rem;
+}
+
+.mod-man-banner {
+    max-width: 60%;
+    max-height: 15rem;
+    margin: 2rem auto 5rem auto;
+    position: relative;
+}
+
+.mod-count-box > p {
+    margin-bottom: 0;
+}
+
+.ts-primary-button {
+    font-size: 14px;
+    padding-left: 1rem;
+    padding-right: 1rem;
+    padding-top: 0.5rem;
+    padding-bottom: 0.5rem;
+    font-weight: bold;
+    background-color: #00d2ff;
+    color: #000001;
+}
+
+.ts-secondary-button {
+    font-size: 14px;
+    padding-left: 3%;
+    padding-right: 3%;
+    padding-top: 1%;
+    padding-bottom: 1%;
+    font-weight: bold;
+    background-color: #00000100;
+    border-width: 0;
+}
+
+.ts-card {
+    position: relative;
+    font-size: 16px;
+    overflow: hidden;
+    width: 12rem;
+    height: 20rem;
+    flex-grow: 1;
+    border-radius: 0.4rem !important;
+    object-fit: cover;
```

Comment:
Not needed if using `background-image` as there isn't any object to fit. Obviously not a blocker, but a note for cleaning up the css.